Directory.Build.targets

```diff
@@ -0,0 +1,16 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
+    <PackageReference Include="GitInfo" Version="*" PrivateAssets="all" />
```

Comment:
should we use an explicit version rather than a floating version for GitInfo here?

Reply:
Probably, yes.